platform/android/scripts/trigger-external-deploy-steps.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+SDK_FLAVOR=${SDK_FLAVOR:-"maps"}
+
+step "Triggering automated site and documentation generation for ${SDK_FLAVOR} SDK ${VERSION_TAG}"
+
+# No branch specified, so default branch will be used
+request_body="{
+  \"request\": {
+    \"message\": \"[${SDK_FLAVOR}] ${VERSION_TAG} automated site and documentation generation\",
+    \"config\": {
+      \"merge_mode\": \"deep_merge\",
+      \"env\": {
+        \"SDK_FLAVOR\": \"${SDK_FLAVOR}\",
+        \"RELEASE_TAG\": \"${VERSION_TAG}\"
+      }
+    }
+  }
+}"
+
+step "Making request…"
+
+# Request URL set to docs-sandbox while testing
+curl -s -X POST \
+  -H "Content-Type: application/json" \
+  -H "Accept: application/json" \
+  -H "Travis-API-Version: 3" \
+  -H "Authorization: token ${TRAVISCI_API_TOKEN}" \
+  -d "${request_body}" \
+  https://api.travis-ci.com/repo/mapbox%2Fdocs-sandbox/requests
```

Comment:
Instead of copying this script, move the original in `platform/ios/scripts` to `scripts/` and make the repo an additional parameter/environment variable.

Reply:
Great call, @friedbunny. Made that chance in my last commit 👍 

Looks like travis requests are being formulated in circle and I can see them reflected in Travis' logs on the other end.